src/Symfony/Component/DependencyInjection/Tests/ContainerTest.php

```diff
@@ -198,6 +197,23 @@ public function testGetThrowServiceNotFoundException()
         }
     }
 
+    public function testGetThrowServiceNotFoundExceptionWithCorrectAlternatives()
+    {
+        $sc = new ProjectServiceContainer();
+        $sc->set('internal_lookalike', $foo = new \stdClass());
+        $privates = new \ReflectionProperty(ProjectServiceContainer::class, 'privates');
+        $privates->setAccessible(true);
+        $this->assertArrayHasKey('internal', $privates->getValue($sc));
+
+        try {
+            $sc->get('int');
+            $this->fail('->get() throws an Symfony\Component\DependencyInjection\Exception\ServiceNotFoundException if the key does not exist');
+        } catch (\Exception $e) {
```

Comment:
IMHO, unless there is a good reason, new tests should use the annotations.
When older tests where introduced, I guess those annotations (as well as `TestCase::setExpectedException()`) were not available with the PHPUnit version in use :)